LIMS_IMAGE/web/templates/laboratory/distribution.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'base.html' %} 
+{% load static %}
+{% load bootstrap5 %} 
+{% load admin_urls %}
+{% block title %}Sample List{% endblock %}
+{% block additionalStyles %}
+<link rel="stylesheet" href="{% static 'laboratory/css/distribution.css' %}" />
+<script src="{% static 'laboratory/js/sample_list_search.js' %}" defer></script>
+{% endblock %}
+<body>
+  {% block content %}
+    <div class="container">
+      {% if samples|length is 0 %}
+      <p>No samples found</p>
+      {% else %}
+      <h1>Samples Ready for Distribution</h1> 
+        <table>
+          <tr>
+            <th>Sample No.</th>
+            <th>Sample Type</th>
+            <th>Sample Form</th>
+            <th>Test Code (Appendix-C)</th>
+            <th>SOP No.</th>
+            <th>Lab Personnel</th>
+          </tr>
+          {% for sample in samples %}
+          {% comment %} samples = context = {'samples': sample_list} {% endcomment %}
+            <tr>
+              <td> <a href="{% url 'laboratory:view_sample' sample.id %}">{{ sample.id }}</a></td>
+              <td>{{ sample.sample_type }}</td>
+              <td>{{ sample.sample_form }}</td>
+              <td></td>
+              <td>{{ sample.sop_number }}</td>
+              <td> {{ sample.lab_personel }}</td>
+            </tr>
+          {% endfor %}
+        </table>
+
+        <h1>Lab Breakdown</h1> 
+        <table>
+          <tr>
+            <th>Sample No.</th>
+            <th>Assigned Lab</th>
+          </tr>
+          {% for ls in labsamples %}
```

Comment:
This was in the distribution document from the client. Should I remove it? I get that it doesn't exactly make a ton of sense here, but was just trying to follow their document.

Reply:
I see what you mean.
I was interpreting that as "this is info we want stored" not necessarily "this is info we want displayed"
I think it doesn't really make sense here, and we should remove it. The info accessible from the lab/test sample page anyways.
If you'd rather keep it though, since it is on the doc, we can keep it

Reply:
Okay I'll get rid of them.